util/src/main/java/com/pkb/pkbcommon/ageutil/AgeUtilService.java

```diff
@@ -0,0 +1,25 @@
+package com.pkb.pkbcommon.ageutil;
+
+import com.pkb.common.datetime.DateTimeService;
+
+import java.time.LocalDate;
+import java.time.ZoneOffset;
+import java.time.temporal.ChronoUnit;
+
+public class AgeUtilService {
+
+    private final DateTimeService dateTimeService;
+
+    public AgeUtilService(DateTimeService dateTimeService) {
+        this.dateTimeService = dateTimeService;
+    }
+
+    public final static int MINIMUM_AGE_LETTER_INVITATION = 16;
+
+    public boolean patientIsOlderThan(LocalDate dateOfBirth, int ageInYears) {
```

Comment:
NB: Given this is date-time related, why not put it in the date-time module rather than a new util package?

Reply:
Hmm I suppose my thinking was that the date-time module was specifically for DateTimeService-related stuff, but actually I could have put this there, you're right. I will probably move it

Reply:
I think I'm going to leave this in the util module, for now.